skopt/utils.py

```diff
@@ -388,3 +391,62 @@ def point_aslist(search_space, point_as_dict):
     ]
     return point_as_list
 
+
+def normalize_dimensions(dimensions, base_estimator="GP"):
+    """
```

Comment:
@MechCoder we would then remove the `base_estimator="GP"` argument, the checks for expected strings associated with that argument, and the check for the regressor, right? 

Basically, we would keep lines 432-449?